modules/scripts/src/base/collections.ts

```diff
@@ -1,23 +1,18 @@
-/// <reference path="../../../../scripts/bin/components/components.d.ts" />
-/// <reference path="../../../../scripts/bin/components/modules.d.ts" />
+import {BoundConstraintValidator, Descriptor, Map} from "../../../../scripts/bin/app/modules.api";
```

Comment:
maybe change `Map` to `FugaziMap` (or something like that) so there won't be a mix-up with the native `Map` object